ch_tools/chadmin/cli/table_metadata.py

```diff
@@ -0,0 +1,93 @@
+import re
+import uuid
+
+UUID_PATTERN = "UUID"
+OFFSET_UUID_PATTERN = 1
+ENGINE_PATTERN = "ENGINE"
+
+PARSER_SUPPORT_TABLE_ENGINE = [
+    "MergeTree",
+    "ReplacingMergeTree",
+    "SummingMergeTree",
+    "AggregatingMergeTree",
+    "CollapsingMergeTree",
+    "VersionedCollapsingMergeTree",
+    "ReplicatedMergeTree",
+]
+
+
+class TableMetadata:
+    @property
+    def table_uuid(self):
+        return self._table_uuid
+
+    @property
+    def table_engine(self):
+        return self._table_engine
+
+    def __init__(self, table_metadata_path: str):
+        assert table_metadata_path.endswith(".sql")
+
+        self._table_uuid = None
+        self._table_engine = None
+
+        with open(table_metadata_path, "r", encoding="utf-8") as metadata_file:
+            for line in metadata_file:
+                if UUID_PATTERN in line:
+                    self._table_uuid = TableMetadata._parse_uuid(line)
+                    continue
+                if ENGINE_PATTERN in line:
+                    result = TableMetadata._parse_engine(line)
+                    self._table_engine = result
+                    break
```

Comment:
Why? We stable format.

Reply:
Because it makes logic more complicated. We have to alway keep in mind what is the order. 
Also what if we will want several keys from single line and both of them are optional?
```
... (optional) key1 value1 .... (optional) key2 value2
``` 
Such case will be complicated to handle correctly with `continue - break` logic

Reply:
Its doesnt matter right now. I will remove order so we will descuss about it later.